.test-infra/jenkins/verify_performance_test_results.py

```diff
@@ -0,0 +1,252 @@
+    else:
+        print("This type is not supported")
+        return None
+
+def count_queries(table_name, time_start, time_stop, metric):
+    # This function checks how many data was inserted in time interval.
+    sql_command = 'select count(*) as query_result from {} where TIMESTAMP > {} and TIMESTAMP < {} and METRIC=\'{}\''.format(
+        table_name,
+        time_start,
+        time_stop,
+        metric
+        )
+    count = submit_big_query_job(sql_command, "value")
+    print("Number of records inserted into {} between {} - {}: {}".format(
+        table_name,
+        time.strftime(TIME_PATTERN, time.gmtime(time_start)),
+        time.strftime(TIME_PATTERN, time.gmtime(time_stop)),
+        count))
+    return count
+
+def get_average_from(table_name, time_start, time_stop, metric):
+    # This function return average value of the provided metric in time interval.
+    sql_command = 'select avg(value) as query_result from {} where TIMESTAMP > {} and TIMESTAMP < {} and METRIC=\'{}\''.format(
+        table_name,
+        time_start,
+        time_stop,
+        metric
+    )
+    average = submit_big_query_job(sql_command, "value")
+    return average
+
+def get_stddev_from(table_name, time_start, time_stop, metric):
+    # This function return standard deviation of the provided metric in time interval.
+    sql_command = 'select stddev(value) as query_result from {} where TIMESTAMP > {} and TIMESTAMP < {} and METRIC=\'{}\''.format(
+        table_name,
+        time_start,
+        time_stop,
+        metric
+    )
+    stddev = submit_big_query_job(sql_command, "value")
+    return stddev
+
+def get_records_from(table_name, time_start, time_stop, metric, number_of_records):
+    # This function checks how many data was inserted in time interval.
+    sql_command = 'select value as query_result from {} where TIMESTAMP > {} and TIMESTAMP < {} and METRIC=\'{}\' ORDER BY TIMESTAMP DESC LIMIT {}'.format(
+        table_name,
+        time_start,
+        time_stop,
+        metric,
+        number_of_records
+        )
+    list_of_records = submit_big_query_job(sql_command, "list")
+    return list_of_records
+
+def calculate_historical_data(bq_table_name, TIME_POINT_2, TIME_POINT_1, metric, nb_older_records, average_recent):
+    average_old = get_average_from(bq_table_name, TIME_POINT_2, TIME_POINT_1, metric)
+    if nb_older_records > 1:
+        stddev_old = get_stddev_from(bq_table_name, TIME_POINT_2, TIME_POINT_1, metric)
+    else:
+        # Standard deviation is 0 when there is only single value.
+        stddev_old = 0
+    percentage_change = 100*(average_recent - average_old)/average_old
+    return percentage_change, stddev_old
+
+def update_messages(bq_table_name, metric, average_recent, percentage_change, stddev_recent, stddev_old,
+                    report_message, slack_report_message):
+    report_message = report_message + '{} - {}, avg_time {:.2f}s, change {:+.3f}%, stddev {:.2f}, stddev_old {:.2f} \n'.format(
```

Comment:
Please update these messages to that it's clear whether the change is a improvement or a reduction in performance.